back/app/services/insights/common_ground_phase_insights_service.rb

```diff
@@ -0,0 +1,49 @@
+module Insights
+  class CommonGroundPhaseInsightsService < IdeationPhaseInsightsService
+    private
+
+    def phase_participations
+      # Events are not associated with phase, so attending_event not included at phase-level.
+      {
+        posting_idea: participation_ideas_published,
+        reacting_idea: participation_idea_reactions
+      }
+    end
+
+    def participation_ideas_published
```

Comment:
Is this the same as ideation? If so then do we need it given that we're already inheriting from `IdeationPhaseInsightsService`?